Sources/BSON/ExtendedJSONEncoder.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
```

Comment:
nit: add a line break between the import statement and the docstring

Reply:
done